src/lib/constants.ts

```diff
@@ -9,25 +10,46 @@ export const exitWithMessage = (message: string): void => {
   process.exit(0);
 };
 
+/**
+ * Logs an array of key-value pairs in tabular format using the 'Table' class and outputs it to the console.
+ *
+ * @param {Array<Record<string, string>>} keyValues - An array of key-value pairs represented as objects.
+ * Each object in the array should have string properties for keys and values.
+ * Example: [{ key1: value1 }, { key2: value2 }, ...]
+ * @returns {void} - This function does not return any value; it logs the table directly to the console.
+ */
+export const logTable = (keyValues: Array<Record<string, string>>): void => {
+  const table = new Table();
+  keyValues.forEach((kv) => table.push(kv));
+  console.log(table.toString());
+};
+
 export const strings = {
   ABORTED: error('Aborted.'),
   BUILD_FOLDER: success(`Build Folder not detected. \nPlease enter the name of the build folder:`),
   BUILD_METADATA_PROMPT: `Select your ${chalk.blue('Build metadata')} file:`,
+  CANNOT_FIND_REPO: (repoName: string) => error(`Cannot find repo ${repoName}`),
```

Comment:
When does this come up?

Reply:
if the user passes the repo without going through the prompt but the repo is not found